secret-provider/src/main/scala/io/lenses/connect/secrets/config/AWSProviderSettings.scala

```diff
@@ -23,6 +23,7 @@ case class AWSProviderSettings(
   fileWriterOpts:   Option[FileWriterOptions],
   defaultTtl:       Option[Duration],
   endpointOverride: Option[String],
+  altRegion:        String
```

Comment:
`altRegion` would need to be made an Option so as not to cause existing users to have to change their settings